igcollect/logfile_values.py

```diff
@@ -259,7 +259,7 @@ def convert_to_timestamp(time_str, time_format):
         if time_format.endswith('z') and time_str[-3] == ':':
             time_str = ''.join(time_str.rsplit(':', 1))
 
-        dt = datetime.datetime.strptime(time_str, time_format).utctimetuple()
+        dt = datetime.datetime.strptime(time_str, time_format).timetuple()
```

Comment:
I tested around a bit. And I still see problems with `timetuple` if the timezone is not specified, which could happen if `time_format` does not end with `z`.
We need to raise the question, what should happen if `strptime` does not return a datetime with a timezone.
Furthermore, I also see less problems with just using the `timestamp` method of the `datetime` object.
A solution could be to check if the datetime has a timezone, if not, we set the timezone explicitly to `utc` and then just using the `timestamp` method of the datetime object.